lib/Db/CircleRequest.php

```diff
@@ -184,6 +184,7 @@ public function getCircles(?IFederatedUser $initiator, CircleProbe $probe): arra
 		$qb->filterCircles(CoreQueryBuilder::CIRCLE, $probe);
 		if (!is_null($initiator)) {
 			$qb->limitToInitiator(CoreQueryBuilder::CIRCLE, $initiator);
+			$qb->orderBy($qb->generateAlias(CoreQueryBuilder::CIRCLE, CoreQueryBuilder::INITIATOR) . '.level', 'desc');
```

Comment:
This doesn't seems to work for the front, I still have to sort https://github.com/nextcloud/contacts/blob/19f836ee23f726b919134c6642af5b8983df871e/src/components/AppNavigation/RootNavigation.vue#L351-L356

Reply:
Do you have cache ? can you try (with the right userId) :
>    ./occ circles:manage:list --initiator-type user --initiator <userId>

This should returns the same list than the ocs route GET /circles

Reply:
| Back | Front |
|:---------:|:------:|
|![image](https://github.com/nextcloud/circles/assets/14975046/98ddf02c-0445-463b-859c-a113d62854ca)|![image](https://github.com/nextcloud/circles/assets/14975046/edf78e55-b057-4978-a033-79518e7d5589)|


https://github.com/nextcloud/contacts/pull/3665